index.html

```diff
@@ -0,0 +1,136 @@
+       <p class="theme h4 pt-5">Unite 100 000 Women in Tech to Foster Innovation with Purpose and Impact</p>
+      </div>
+
+    </main>
+  </section>
+<!-- End of Main Section -->
+<!-- Program Schedule -->
+<section class="program container-fluid">
+  <div class="program-box d-flex flex-column justify-content-center align-items-center">
+    <h2 class="h2 text-center pt-5">Summit Program</h2>
+    <div class="pt-1 underprogram"></div>
+  </div>
+  <div class=" d-md-flex flex-row mt-5 px-4">
+    <div class="program-item pt-2 mx-1 mt-3 px-3">
+      <div class="program-logo"><i class="fa-sharp fa-solid fa-laptop-file"></i></div>
+      <div class="pro-topic"><h3>Works Exhibition</h3></div>
+      <div class="prosub-topic text-light text-center"><p>A number of Women and groups will have a chance to showcase their products and services in various booths.</p></div>
+    </div>
+    <div class="program-item pt-2 mx-1 mt-3 px-3">
+      <div class="program-logo"><i class="fa-regular fa-lightbulb"></i></div>
+      <div class="pro-topic"><h3>Innovative session</h3></div>
+      <div class="prosub-topic text-light text-center"><p>The session will be a practical implementation of ideas that result in the introduction of new goods or services or improvement in offering goods or services.</p></div>
+    </div>
+    <div class="program-item pt-2 mx-1 mt-3 px-3">
+      <div class="program-logo"><i class="fa-regular fa-gift"></i></div>
+      <div class="pro-topic"><h3>Award Presentation</h3></div>
+      <div class="prosub-topic text-light text-center"><p>Participants will get a chance to present their portfolio to sponsors. The winner with the best portfolio will also get $1000 cash price and instant employment</p></div>
+    </div>
+    <div class="program-item pt-2 mx-1 mt-3 px-3">
+      <div class="program-logo"><i class="fa-regular fa-people-line"></i></div>
+      <div class="pro-topic"><h3>Interactive Lecture</h3></div>
+      <div class="prosub-topic text-light text-center"><p>Interactive session with leading women in Tech. The Lecture will be conducted in large groups.</p></div>
+    </div>
+    <div class="program-item pt-2 mt-3 px-3">
+      <div class="program-logo"><i class="fa-solid fa-handshake"></i></div>
+      <div class="pro-topic"><h3>Networking</h3></div>
+      <div class="prosub-topic text-light text-center"><p>Participants will get a chance to connect and network with other women in different tech communities during this event. This will be a great opportunity build relations.</p></div>
+    </div>
+  </div>
+    <div class="program-bo d-flex flex-column justify-content-center align-items-center">
+      <h2 class="h4 text-center prog-see py-5"><a href="#" class="text-dark">See more Programs</a></h2>
+    </div>
+    <div class="join container align-items-center justify-content-center px-3"><a href="#" class="joina"> <h3 class="text-center text-white">Join Women In Tech Summit</h3></a> </div>
+</section>
```

Comment:
- [ ] As request by the previous reviewer, kindly make the background color of this section dark to match the design guidelines.
### Your design
![1](https://user-images.githubusercontent.com/94551484/198362243-8732bc7d-0898-4a4b-9be6-8408551db66c.png)

### Design guidelines
![f1](https://user-images.githubusercontent.com/94551484/198362280-19f4e349-7051-46b1-962f-01f30f2045fd.png)

Reply:
This is a background image and not a color and if I remember correctly from the program, it says we can change the images it wants. But I have changed it to black as you said.